views/pages/saved_matches.ejs

```diff
@@ -28,7 +28,7 @@
           <input value="<%= data[i]._id%>" type="hidden" name="userId" />
           <button type="submit" class="remove">Remove</button>
         </form>
-        <a href="/chat:id">Chat!</a>
+        <a href="/chat/1">Chat!</a>
```

Comment:
Why do you use a number here instead of :id?

Reply:
This was just for a test. Should be `/chat/<%= chatNumber %>` when it's totally working